packages/components/organisms/f-form/src/components/_tests/Form.test.js

```diff
@@ -76,17 +96,205 @@ describe('Form', () => {
             });
         });
 
-        describe('onFormSubmit :: ', () => {
-            it('should emit `form-submitting`', () => {
+        describe('isFormValid ::', () => {
+            let touchSpy;
+
+            beforeEach(() => {
+                touchSpy = jest.spyOn($v, '$touch');
+            });
+
+            afterEach(() => {
+                jest.clearAllMocks();
+            });
+
+            it('should call `.touch()` ', () => {
+                // Act
+                const wrapper = shallowMount(FForm, { propsData, mocks: { $v } });
+
+                wrapper.vm.isFormValid();
+
+                // Assert
+                expect(touchSpy).toBeCalled();
+            });
+
+            it('should return `true` if `$v` is valid', () => {
                 // Arrange
-                const wrapper = shallowMount(FForm, { propsData });
+                $v.$invalid = false;
+
+                // Act
+                const wrapper = shallowMount(FForm, { propsData, mocks: { $v } });
+
+                // Assert
+                expect(wrapper.vm.isFormValid()).toBeTruthy();
+            });
+
+            it('should return `false` if `$v` is invalid', () => {
+                // Arrange
+                $v.$invalid = true;
+
+                // Act
+                const wrapper = shallowMount(FForm, { propsData, mocks: { $v } });
```

Comment:
Could setting `$invalid` like this potentially carry state across tests? I wonder if something like this might be a bit more robust:

(Feel free to trim it down a bit visually if you think it takes up too many lines)

```suggestion
                // Arrange & Act
                const wrapper = shallowMount(FForm, {
                    propsData,
                    mocks: {
                        $v: {
                            ...$v,
                            $invalid: true
                        }
                    }
                });
```